docs/Build/wallet.md

```diff
@@ -18,67 +18,79 @@ cd cardano-wallet
 
 ##### Build Cardano Wallet
 
-You can use the instructions below to build the cardano-node, same steps can be executed in future to update the node (replacing appropriate tag) as well.
+You can use the instructions below to build the latest release of [cardano-wallet](https://github.com/input-output-hk/cardano-wallet).
+
+!> - Note that the latest release of `cardano-wallet` may not work with the latest release of `cardano-node`. Please check the compatibility of each `cardano-wallet` release yourself in the official docs, e.g. https://github.com/input-output-hk/cardano-wallet/releases/tag/v2021-05-26.
```

Comment:
We could use https://github.com/input-output-hk/cardano-wallet/releases/latest instead of the `tag/v2021-05-26`

Reply:
> Until the component is bought to the same 'visibility' as other IOG components, I fear we'd have to stick with master - so that's good

Hehe, yeah that was my feeling as well.